src/main/java/edu/harvard/iq/dataverse/confirmemail/ConfirmEmailServiceBean.java

```diff
@@ -102,11 +108,20 @@ private void sendLinkOnEmailChange(AuthenticatedUser aUser, String confirmationU
 
         try {
             String toAddress = aUser.getEmail();
-            /**
-             * @todo Move this to Bundle.properties.
-             */
-            String subject = BundleUtil.getStringFromBundle("notification.email.verifyEmail.subject");
-            mailService.sendSystemEmail(toAddress, subject, messageBody);
+            try {
+                Dataverse rootDataverse = dataverseService.findRootDataverse();
+                if (rootDataverse != null) {
+                    String rootDataverseName = rootDataverse.getName();
+                    // FIXME: consider refactoring this into MailServiceBean.sendNotificationEmail. Then we might need
+                    UserNotification userNotification = new UserNotification();
+                    userNotification.setType(UserNotification.Type.CONFIRMEMAIL);
+                    String subject = MailUtil.getSubjectTextBasedOnNotification(userNotification, rootDataverseName);
+                    logger.fine("sending email to " + toAddress + " with this subject: " + subject);
+                    mailService.sendSystemEmail(toAddress, subject, messageBody);
+                }
+            } catch (Exception e) {
+                logger.info("The root dataverse is not present. Don't send a notification to dataverseAdmin.");
```

Comment:
Meh, as we discussed in person, anything higher than "fine" is shown. I think "info" is high enough.